openmetadata-ui/src/main/resources/ui/src/components/Database/SchemaEditor/SchemaEditor.tsx

```diff
@@ -118,7 +118,7 @@ const SchemaEditor = ({
         value={internalValue}
         onBeforeChange={handleEditorInputBeforeChange}
         onChange={handleEditorInputChange}
-        onFocus={onFocus}
+        {...(onFocus && { onFocus })}
```

Comment:
@karanh37 can you explain this change please?

Reply:
The `CodeMirror` component was throwing an error when the onFocus prop was undefined. Since onFocus is conditionally passed, added this fix.